lib/screen/scan_qr/scan_qr_page.dart

```diff
@@ -569,7 +569,10 @@ class QRScanViewState extends State<QRScanView>
   }
 
   void _onQRViewCreated(QRViewController controller) {
-    this.controller = controller;
+    // Dispose of the previous controller if it exists
+    _controller?.dispose();
```

Comment:
we dont need to dispose controller here

Reply:
if we don't dispose it and assign new value, it would cause memory leak